docker/build_artifacts/sagemaker/serve.py

```diff
@@ -324,6 +327,24 @@ def _wait_for_gunicorn(self):
                 log.info("gunicorn server is ready!")
                 return
 
+    def _wait_for_tfs(self):
+        # Wait until tfs server is up and running
+        while True:
+            try:
+                tfs_ready_count = 0
+                for i in range(self._tfs_instance_count):
+                    tfs_url = "http://localhost:{}/v1/models/{}/metadata" \
+                        .format(self._tfs_rest_port[i], self._tfs_default_model_name)
+                    log.info("Trying to connect with model server \n {}".format(tfs_url))
+                    response = requests.get(tfs_url)
+                    logging.info(response)
```

Comment:
Unless response already includes server/endpoint metadata please consider logging some additional information to help customers identify which server returned which response.